client-v2/src/main/java/com/clickhouse/client/api/metadata/TableSchema.java

```diff
@@ -16,6 +16,8 @@ public class TableSchema {
 
     private List<ClickHouseColumn> columns;
 
+    private List<ClickHouseColumn> columnsView;

```

Comment:
columnsView is just a read-only view. Previous code was wrapping each time getColumns() is called. 
It is enough to create view once (because it is a view)